plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -74,6 +76,7 @@
 public class BigQueryClient
 {
     private static final Logger log = Logger.get(BigQueryClient.class);
+    private static final String TRINO_QUERY_LABEL = "trino_query_id";
```

Comment:
Nit: Using a private constant with an obvious value (as opposed to some random looking number where the name gives context) makes the code less readable, since you have to look in another place to find the value. And it's only used in one place, so inlining it would be more readable.